ckanext/dcat/profiles.py

```diff
@@ -1209,9 +1217,6 @@ def graph_from_dataset(self, dataset_dict, dataset_ref):
 
         # Basic fields
         items = [
-            ('title', DCT.title, None, Literal),
```

Comment:
In the `grap_from_dataset()` case we want to do the opposite, translate the `_translated` fields into whatever triples are needed to express multilingual metadata. At the very least I would keep these lines to provide a default fallback for the serialization of these fields, and then if necessary, handle the `_translated` ones

Reply:
N/A `profiles.py` reverted.